app/(dashboard)/dashboard/participants/_components/ImportCSVModal.tsx

```diff
@@ -129,6 +129,10 @@ const ImportCSVModal = () => {
       });
     }
 
+    if (typeof onUploaded === 'function') {
+      onUploaded();
+    }
+
```

Comment:
🤨 

- The component's verb is _import_, but you've decided to call the callback "uploaded". Let's make it `onImportComplete` or something.
- You made the callback optional with a generic function signature when it is defined. Optional properties are `null` by default, but you are instead testing if it is a function. You can just do `if (onImportComplete)`, or better yet you can use optional chaining and just call it if it exists: `onImportComplete?.()`.

Reply:
resolved here [4060004](https://github.com/complexdatacollective/Fresco/pull/27/commits/4060004f14b15c1fe24651053c902dffb29019dd)

There was a similar pattern in ProtocolUploader that I fixed here as well.